### PR TITLE
feat: create profile qualification

### PR DESCRIPTION
**Pull request by Simpsonkk**
## Describe your changes

- I did the profile qualification step
- realized adding, deleting, creating certificates
- created a stepper for profile steps

Update: I changed the font size from 24 to 16 pixels. The screenshot is in the comments.

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://homecareaid.atlassian.net/browse/AC-93?atlOrigin=eyJpIjoiYWY1ZTYzMWI0Y2FjNDc5MGFlZDkyYzkzYWFjYmIyYjIiLCJwIjoiaiJ9)

image
image
image
image

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [ ] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [ ] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

### CODE REVIEW

File: src/components/profile/profile-qualification/certificate-list/CertificateList.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+          setCertificates={setCertificates}
+        />
+      </Modal>
+
+      <Modal
+        isActive={isDeleteModalActive}
+        onClose={onDeleteModalClose}
+        title={translate('profileQualification.deleteCertificate')}
+      >
+        <Box display="flex" flexDirection="column">
```
Comment (nadiia-tsytsylina) [inline src/components/profile/profile-qualification/certificate-list/CertificateList.tsx:117]: We don't use inline styles

File: src/components/profile/profile-qualification/certificate-list/CertificateList.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+          <Title>{translate('profileQualification.deleteWarning')}</Title>
+
+          <Box display="flex" gap={2}>
```
Comment (nadiia-tsytsylina) [inline src/components/profile/profile-qualification/certificate-list/CertificateList.tsx:120]: here too

**Review by Lilly-nn — APPROVED** [review]

**Review by nadiia-tsytsylina — APPROVED** [review]

**Comment by lyudad** [comment]
Screenshot 2023-11-24 at 20 40 14

**Comment by Simpsonkk** [comment]
> Screenshot 2023-11-24 at 20 40 14

I made a font size less

image

**Review by fedasevich — APPROVED** [review]

File: src/components/profile/profile-qualification/certificate-form/constants.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { ProfileQualityFormValues } from 'src/components/profile/profile-qualification/types';
+
+export const DATE_FORMAT = 'dd/MM/yyyy';
```
Comment (lyudad) [inline src/components/profile/profile-qualification/certificate-form/constants.ts:3]: не согласна. формат даты должен быть один на проект, а значить константы не компонента а проекта

File: src/components/profile/profile-qualification/certificate-form/constants.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+export const URL_PATTERN = /^https:\/\/.+$/;
```
Comment (lyudad) [inline src/components/profile/profile-qualification/certificate-form/constants.ts:5]: тут в принципе тоже самое

File: src/components/profile/profile-qualification/certificate-form/constants.ts (hunk 3 above)
Comment (Simpsonkk) [reply src/components/profile/profile-qualification/certificate-form/constants.ts]: done

File: src/components/profile/profile-qualification/certificate-form/constants.ts (hunk 4 above)
Comment (Simpsonkk) [reply src/components/profile/profile-qualification/certificate-form/constants.ts]: done

**Pull request merged**